src/applications/avs/utils/index.js

```diff
@@ -3,23 +3,51 @@ import { format, utcToZonedTime } from 'date-fns-tz';
 import { formatDateLong } from '@department-of-veterans-affairs/platform-utilities/exports';
 
 const parseProblemDateTime = dateString => {
-  // Parse dates in the format "Thu Apr 07 00:00:00 PDT 2005"
-
-  // Extract the month, day, and year
-  const parts = dateString.split(' ');
-  const month = parts[1];
-  const day = parseInt(parts[2], 10);
-  const year = parseInt(parts[5], 10);
+  try {
```

Comment:
use datefns parse with defined format instead

Reply:
datefns can't parse the timezone, so went with a hybrid but more restricted approach